src/entity/update/movement.rs

```diff
@@ -36,7 +37,7 @@ fn resolve_collisions(
         Vector2::new(0.0, 0.0)
     };
     let speed = unitary_speed * entity.stats.speed;
-    let new_position = entity.position + (speed * *SEC_PER_UPDATE);
+    let new_position = entity.position + speed * tick_duration;
```

Comment:
Oh this is surprising. Typically the physics use a separate time step. (typically as in unity) :)